app/managers.py

```diff
@@ -0,0 +1,62 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db")
+        self.create_table()
+        self.table_name = "actors"
+
+    def create_table(self) -> None:
+        self._connection.execute(
+            """
+            CREATE TABLE IF NOT EXISTS actors (
+                id INTEGER PRIMARY KEY AUTOINCREMENT,
+                first_name TEXT NOT NULL,
+                last_name TEXT NOT NULL
+            )
+            """
+        )
+        self._connection.commit()
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f""""
```

Comment:
must be triple quotes here, double check this block again and make sure everything  is aligned